content/zh-cn/docs/reference/kubernetes-api/config-and-storage-resources/persistent-volume-v1.md

```diff
@@ -17,13 +17,15 @@ content_type: "api_reference"
 description: "PersistentVolume (PV) is a storage resource provisioned by an administrator."
 title: "PersistentVolume"
 weight: 5
+auto_generated: true
```

Comment:
The translated version is not auto-generated, so this line can/should be safely dropped.